### ISSUE TITLE
Drag-n-drop doesn't work on touch devices

### ISSUE THREAD

**Issue by angrybrad** [craftcms/cms#19]
#### Description:

All of the drag-n-drop features in the CP are inaccessible on mobile/touch devices where mouse events don't exist.
